docs/TESTS.md

```diff
@@ -1,112 +1,65 @@
-## Xcode Tutorial
+## Testing Tutorial
 
-This guide explains how to create and configure a Swift-based Xcode project for Exercism.
+This guide explains how to run unit tests for Exercism Swift exercises, and optionally how to use Xcode for development.
 
-Apple does not support user-defined project templates in Xcode, and the default templates are more elaborate than those needed for Exercism. This guide bridges that gap.
+Exercism uses [Swift Package Manager](https://github.com/apple/swift-package-manager/tree/master/Documentation) to package files and tests for Swift.
```

Comment:
Absolutely. I appreciate your help on this.  We are working on branch so the changes are not going on master yet. I am going to accept this PR. Feel free to submit a new PR with your changes. Thank you!